tests/test_scheduler.py

```diff
@@ -1,14 +1,15 @@
+from random import shuffle
+import collections
 import shutil
 import tempfile
 import unittest
-import collections
 
 from twisted.internet import defer
-from twisted.trial.unittest import TestCase
+from twisted.trial.unittest import TestCase as TwistedTestCase
```

Comment:
I would not make this change in this pull request, but I think it would make sense to have a separate pull requests to rename all of them in the test suite, to avoid misunderstandings.